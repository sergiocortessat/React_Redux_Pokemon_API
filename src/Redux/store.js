import { configureStore } from '@reduxjs/toolkit';
import pokemonReducer from './Slices/homeFetchSlice';

export default configureStore({
  reducer: {
    pokemon: pokemonReducer,
  },
});
