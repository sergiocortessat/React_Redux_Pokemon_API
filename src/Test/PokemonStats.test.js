import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Stats from '../App/Containers/PokemonStats';
import store from '../Redux/store';

describe('Pokemon stats', () => {
  test('should match with snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Provider store={store}>
            <Stats />
          </Provider>
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
