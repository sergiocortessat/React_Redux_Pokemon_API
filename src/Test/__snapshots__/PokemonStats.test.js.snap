// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon stats should match with snapshot 1`] = `
Array [
  <div
    className="button-home"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      HOME
    </a>
  </div>,
  <div
    className="stat-name"
  >
    <h1>
      #undefined
       - 
    </h1>
  </div>,
  <div
    className="wrapper"
  >
    <div
      className="inner"
    >
      <div
        className="info"
      >
        <h2>
          ID
           
          #
        </h2>
        <h2>
          HEIGHT
           
        </h2>
        <h2>
          WEIGHT
           
        </h2>
      </div>
      <div
        className="description"
      >
        <h2>
          DESCRIPTION
        </h2>
        <h3 />
      </div>
    </div>
    <div
      className="stats-image"
    >
      <img />
    </div>
    <div
      className="inner2 stats"
    >
      <h2>
        POKEMON STATS
      </h2>
      <div />
    </div>
  </div>,
]
`;
